test/mailers/previews/notifier_preview.rb

```diff
@@ -1,7 +1,12 @@
 class NotifierPreview < ActionMailer::Preview
   def digest
-    recipients = User.all.pluck :email
-    options = {recipients: recipients, completed: 0, remaining: 3}
+    options = {
+      recipients: ['vasyllasiak@interlink-ua.com'],
+      completed_todos: {List.last => [Item.first, Item.last]},
```

Comment:
@sergeyradkevich, as this class is just an email preview for the testing purposes, I suppose it will be deleted in #18 (modified issues order).